### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by JuanAlmeida12**
#### What is the purpose of this pull request?
As title says.

#### Types of changes
- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,127 +2,82 @@
...
+| `OrderByTopSaleDESC`   | `String`  | Order by the top items sales            |
+| `OrderByPriceDESC`     | `String`  | Order by the items price *(descendent)* |
+| `OrderByPriceASC`      | `String`  | Order by the items price *(ascendant)*  |
+
+`ScrollEnum`:
+| Value | Type      | Description |
+| ----- | --------- | ----------- |
+| `BY_PAGE`      | `String`  | Scroll item page by page |
+| `ONE_BY_ONE`   | `String`  | Scroll item one by one   |
+
```
Comment (brunojdo) [inline README.md]: This app uses summary props, you need to describe too

File: README.md [hunk 2]
```diff
@@ -2,127 +2,84 @@
...
 | [1.x]    | **Current Release** |  2018-11-08     |                       |             | 2.x
 
+See our [LTS policy](https://github.com/vtex-apps/awesome-io#lts-policy) for more information.
 
-## Continuous Integrations
-
-### Travis CI
+## Table of Contents
+- [Usage](#usage)
+- [API](#api)
```
Comment (theomoura) [inline README.md]: Remove API  block here. SHould be removed to match others template.

File: README.md [hunk 3]
```diff
@@ -2,127 +2,84 @@
...
+  - [Blocks API](#blocks-api)
+    - [Configuration](#configuration)
+  - [Styles API](#styles-api)
+- [Troubleshooting](#troubleshooting)
 
-[![Build Status](https://travis-ci.org/vtex-apps/shelf.svg?branch=master)](https://travis-ci.org/vtex-apps/shelf)
 
 ## Usage
+This app use our store builder with the blocks architecture. To know more about Store Builder [click here.](https://help.vtex.com/en/tutorial/understanding-storebuilder-and-stylesbuilder#structuring-and-configuring-our-store-with-object-object)
```
Comment (theomoura) [inline README.md]: Correct: `This app uses our store... `

File: README.md [hunk 4]
```diff
@@ -2,127 +2,84 @@
...
 
-> 1- Add the dependency in your `manifest.json`
+To use this app you need to import in your dependencies on `manifest.json`.
```
Comment (theomoura) [inline README.md]: Correct: 
`To use this app you need to import it in your dependencies on the `manifest.json` file.

File: README.md [hunk 5]
```diff
@@ -2,127 +2,84 @@
...
-## CSS Classes
-
-```css
-/* ==== '/react/CustomClasses.js' ==== */
-vtex-shelf
-vtex-shelf__title-text
-vtex-shelf__title-content
-vtex-shelf__slide
-```
+This app have an interface that describes what are the rules that the blocks must implement when you want to use the shelf block.
```
Comment (theomoura) [inline README.md]: `This app has an...`

File: README.md [hunk 6]
```diff
@@ -2,127 +2,84 @@
...
+```json
+  "shelf": {
+    "component": "Shelf"
+  },
+  "related-products": {
+    "component": "RelatedProducts"
+  }
 ```
+#### Configuration 
+Through the Storefront you can change the behavior and interface of shelf. But, you also can make adjusts in your theme app, like Dreamstore does.
```
Comment (theomoura) [inline README.md]: `But, you can also make adjustments...`

File: README.md [hunk 7]
```diff
@@ -2,127 +2,84 @@
...
+| Value | Type      | Description |
+| ----- | --------- | ----------- |
+| `OrderByTopSaleDESC`   | `String`  | Order by the top items sales            |
+| `OrderByPriceDESC`     | `String`  | Order by the items price *(descendent)* |
+| `OrderByPriceASC`      | `String`  | Order by the items price *(ascendant)*  |
+
+`ScrollEnum`:
+| Value | Type      | Description |
+| ----- | --------- | ----------- |
+| `BY_PAGE`      | `String`  | Scroll item page by page |
```
Comment (theomoura) [inline README.md]: `Scrolls items, page by page`

File: README.md [hunk 8]
```diff
@@ -2,127 +2,84 @@
...
+| `ONE_BY_ONE`   | `String`  | Scroll item one by one   |
```
Comment (theomoura) [inline README.md]: Same here

File: README.md [hunk 9]
```diff
@@ -2,127 +2,84 @@
 
 ## Description
 
-VTEX App that shows a collection of products.
+VTEX App that shows a collection of products. This is a VTEX app that is used by Dreamstore product.
```
Comment (brunojdo) [inline README.md]: shows a shelf of products (means vitrine), it is more detailed.

File: README.md [hunk 10]
```diff
@@ -2,127 +2,83 @@
...
-      -type: String
-    - buyButtonText // Custom buy button's text
-      -type: String
-    - displayBuyButton // Buy button display types.
-      - Type: String
-      - Default: 'DISPLAY_ALWAYS'
-      - Enum: ['DISPLAY_ALWAYS', 'DISPLAY_ON_HOVER', 'DISPLAY_NONE']
-```
+## Blocks API
+:construction: :construction: :construction:
```
Comment (brunojdo) [inline README.md]: remove

File: README.md (hunk 6 above)
Comment (brunojdo) [reply README.md]: `Through the Storefront, you can change the shelf's behavior and interface. However, you also can make in your theme app, as Dreamstore does.`

File: README.md (hunk 4 above)
Comment (brunojdo) [reply README.md]: I think this is a nit, on manifest.json seems correct too.

File: README.md [hunk 11]
```diff
@@ -2,127 +2,103 @@
...
+You can check if others are passing through similar issues [here](https://github.com/vtex-apps/shelf/issues). Also feel free to [open issues](https://github.com/vtex-apps/shelf/issues/new) or contribute with pull requests.
 
 ## Tests
+:construction: :construction: :construction:
 
-Run the tests with the command
-```
-cd react && yarn test
+### Running Tests
+To run the test suit, type the following in the terminal, inside the folder `react`
```
Comment (theomoura) [inline README.md]: Keep the test section like in `minicart` for conciseness

File: README.md [hunk 12]
```diff
@@ -2,127 +2,89 @@
...
 
-```css
-/* ==== '/react/CustomClasses.js' ==== */
-vtex-shelf
-vtex-shelf__title-text
-vtex-shelf__title-content
-vtex-shelf__slide
-```
+## Styles API
+:construction: :construction: :construction:
```
Comment (brunojdo) [inline README.md]: Now, we have.

File: README.md [hunk 13]
```diff
@@ -2,127 +2,156 @@
 
 ## Description
 
-VTEX App that shows a collection of products.
+VTEX App that shows a shelf of products. This is a VTEX app that is used by Dreamstore product.
```
Comment (vitoriaHeliane) [inline README.md]: It is according the outdated template, please, follow the newest one from [Minicart](https://github.com/vtex-apps/minicart/edit/master/README.md) .

File: README.md [hunk 14]
```diff
@@ -2,127 +2,156 @@
...
+
+:loudspeaker: **Disclaimer:** Don't fork this project, use, contribute, or open issue with your feature request.
 
 ## Release schedule
 | Release  | Status              | Initial Release | Maintenance LTS Start | End-of-life | Dreamstore Compatibility
```
Comment (vitoriaHeliane) [inline README.md]: Store Compatibility :hammer:

File: README.md [hunk 15]
```diff
@@ -2,127 +2,156 @@
```
Comment (vitoriaHeliane) [inline README.md]: Thee title should be `VTEX Shelf`

File: README.md [hunk 16]
```diff
@@ -2,127 +2,156 @@
 
 ## Description
 
-VTEX App that shows a collection of products.
+VTEX App that shows a shelf of products, and this app is used by store theme.
```
Comment (vitoriaHeliane) [inline README.md]: The VTEX shelf app is a store component that shows a collection of products, and this app is used by store theme. :hammer:

**Pull request merged**